src/encoded/tests/test_merge_schemas.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from snovault.schema_utils import load_schema
+from .testing_views import TestingLinkedSchemaField
+
+
+class TestMergedSchemas:
+    """ Class that contains tests that validate $merge refs are correctly resolved within repo
+        and across repos
+    """
+
+    @staticmethod
+    def test_merged_schemas():
+        """ Checks that we can resolve multiple different schema $ref types """
+        schema = load_schema(TestingLinkedSchemaField.schema)
+        props = schema['properties']
+        for key in ['access_key_id', 'file_format', 'title']:
+            assert key in props
+        # check access key id
+        assert props['access_key_id']['comment'] == 'Only admins are allowed to set this value.'
```

Comment:
Is comment really the name we define in our own data model? Probably too late(?), but I so wish it said another word (like `"restriction_description"` so it doesn't look meta-syntactic (and hence invisible to computation).

Unrelated, could the string you're looking for be a named constant so that light edits of the text don't break the tests? Or is the place it exists elsewhere only in the schema's .json file? Since I _think_ it's testing the merge, I would accept as a compromise that the string is the same as the string in the thing it's merging from... In fact, I'm a little surprised that you're not just testing that the tree matches that other tree. At a previous employer, I had a testing routine (and I think unittest has a similar one built into it as a .assertxxx operation) that checks that a given dictionary has at least the parts given by another given dictionary  (ignoring additional other things that might have been inserted).